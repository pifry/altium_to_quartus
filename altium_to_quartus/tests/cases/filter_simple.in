NODENAME SIMPLE_NET$
    J3        53  U3      AA23
NODENAME ANOTHER_NET$
    J3        53  U3      AA24