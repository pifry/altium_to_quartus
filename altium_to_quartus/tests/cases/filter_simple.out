set_location_assignment PIN_AA23 -to "SIMPLE_NET"
